README.md

```diff
@@ -1,30 +1,30 @@
 # Mongo Kubernetes Replica Set Sidecar
 
-This project is as a PoC to setup a mongo replica set using Kubernetes. It should handle resizing of any type and be 
+This project is as a PoC to setup a mongo replica set using Kubernetes. It should handle resizing of any type and be
 resilient to the various conditions both mongo and kubernetes can find themselves in.
 
 ## How to use it
 
-The docker image is hosted on docker hub and can be found here:  
```

Comment:
All of these trailing double spaces were to introduce line breaks... i.e. compare
https://github.com/mwuertinger/mongo-k8s-sidecar/blob/remove-trailing-spaces/README.md
with
https://github.com/cvallance/mongo-k8s-sidecar/blob/master/README.md